tutorials/git-basics.rst

```diff
@@ -0,0 +1,909 @@
+    Unpacking objects: 100% (3/3), 312 bytes | 20.00 KiB/s, done.
+    From git@github.com:GITHUB_USERNAME/uchicago-cs-git-tutorial.git
+       e3f9ef1..5716877  main       -> origin/main
+    Updating e3f9ef1..5716877
+    Fast-forward
+     test.txt | 3 ++-
+     1 file changed, 2 insertions(+), 1 deletion(-)
+
+
+If you have multiple local repositories (e.g., one on a CS machine and
+one on your laptop), it is very important that you remember to run
+``git pull`` before you start working, and that you ``git push`` any
+changes you make. Otherwise, your local repositories (and the repository
+on GitHub) may *diverge* leading to a messy situation called a *merge
+conflict* (we discuss conflicts in more detail below). This will be
+specially important once you start using Git for its intended purpose:
+to collaborate with multiple developers, where each developer will have
+their own local repository, and it will become easier for some
+developers’ code to diverge from others’.
+
+Discarding changes and unstaging
+--------------------------------
+
+One of the benefits of using a version control system is that it is very
+easy to inspect the history of changes to a given file, as well as to
+undo changes we did not intend to make.
+
+For example, edit ``test.txt`` to remove all its contents. Make sure you
+do this in your home repository (``/home/USER/git-tutorial/``)
+and not in the temp repository you created earlier.
+
+``git status`` will tell us this:
+
+::
+
+    On branch main
+    Your branch is up to date with 'origin/main'.
+
+    Changes not staged for commit:
+      (use "git add <file>..." to update what will be committed)
+      (use "git restore <file>..." to discard changes in working directory)
+        modified:   test.txt
+
+
+If we want to discard the changes we made to ``test.txt``, all we have
+to do is follow the helpful advice provided by the above output:
+
+::
+
+   git restore test.txt
+
+.. note::
+
+   In older versions of Git, ``git status`` may refer to the ``git checkout`` command.
+   In that case, run this command instead::
+
+        git checkout -- test.txt
+
+If you open ``test.txt``, you’ll see that its contents have been
+magically restored!
+
+Now, edit ``test.txt`` and ``README.md`` to add an additional line with
+the text ``Hopefully our last change...``. Run ``git add -u`` but don’t
+commit it just yet. ``git status`` will show this:
+
+::
+
+    On branch main
+    Your branch is up to date with 'origin/main'.
+
+    Changes to be committed:
+      (use "git restore --staged <file>..." to unstage)
+        modified:   README.md
+        modified:   test.txt
+
+
+Now, let’s say we realized we want to commit the changes to
+``README.md``, but not to ``test.txt``. However, we’ve already told git
+that we want to include ``test.txt`` in the commit. Fortunately, we can
+“un-include” it (or “unstage” it, in Git lingo) by running this:
+
+::
+
+   git restore --staged test.txt
+
+.. note::
```

Comment:
Did you meant `git restore` instead of `git status`?

Reply:
Same as previous comment; the note refers to the output from `git status`